en/python_installation/instructions.md

```diff
@@ -9,9 +9,10 @@ Please install normal Python as follows, even when you have Anaconda installed o
 <!--sec data-title="Install Python: Windows" data-id="python_windows" data-collapse=true ces-->
 
 First check whether your computer is running a 32-bit version or a 64-bit version of Windows, on the "System type" line of the System Info page. To reach this page, try one of these methods:
-* Press the Windows key and Pause/Break key at the same time
+* Press the Windows key and start to type `System Information`. The menu should bring it up as best match within 3 letters. Click on it or press enter to accept it.
```

Comment:
This is not very clear to me. I know what you mean, but not sure how to phrase it briefly. It also feels like the most complex of all so should be last IMHO.

Reply:
Would this be better?

> * Search the Windows Start menu for "System Information". To do that, click the Start button or press the Windows key, then begin to type `System Information`. You'll probably be able to select that entry without having to type it all out completely.

Reply:
`You'll probably be able to select that entry without having to type it all out completely.` isn't that clear. How about:
> It will start making suggestions as soon as you type. You can select the entry once it shows up.

Reply:
So that'd be
```suggestion
* Search the Windows Start menu for "System Information". To do that, click the Start button or press the Windows key, then begin to type `System Information`. It will start making suggestions as soon as you type. You can select the entry once it shows up.
```

Can we make that shorter without mutilating it? Or is the length alright? (Maybe less of an issue, if it isn't the first option listed.)

Reply:
Length is IMHO alright if it's not the first option.